pkg/core/curves/native/bls12-381/arithmetic_fallback.go

```diff
@@ -158,26 +158,15 @@ func subAssign(z, x *fe) {
 }
 
 func updateIfBNotZero(z *fe, b uint64) {
-	// Get cmove flag based off of b
```

Comment:
We need the constant time here. Is this really the problem? I believe the real issue is in the header that you addressed. We don't want to undo the constant time code with the rest of this PR.

Reply:
@mikelodder7 I tried with just the header but these tests inside `arithmetic_fallback_test.go` were failing:

- TestFp2AdditionPropertiesAssigned
    fp_test.go:580: 0-a == -a
- TestFp2SquareRoot
    fp_test.go:812: square root failed
- TestFp6AdditionPropertiesAssigned
    fp_test.go:963: 0-a == -a
- TestFp6Inversion
    fp_test.go:1166: (r*a) * r*(a^-1) == r)
- TestFp12Inversion
    fp_test.go:1389: (r*a) * r*(a^-1) == r)
- TestPairingExpected
    pairing_test.go:36: invalid element
- TestPairingNonDegeneracy
    pairing_test.go:67: pairing result is not valid
- TestPairingBilinearity
    pairing_test.go:190: bad pairing, 2
- TestPairingMulti
    pairing_test.go:248: fail multi pairing